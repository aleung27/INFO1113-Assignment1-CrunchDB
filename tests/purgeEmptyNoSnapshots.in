PURGE a
PURGE b
SET a 4
SET K 0
PURGE a
LIST ENTRIES
PURGE K
LIST ENTRIES
PURGE a
