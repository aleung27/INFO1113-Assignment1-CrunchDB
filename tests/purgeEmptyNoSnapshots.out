> ok

> ok

> ok

> ok

> ok

> K [0]

> ok

> no entries

> ok

